apps/backend-e2e/src/e2e/project.cy.ts

```diff
@@ -526,72 +490,69 @@ describe('PROJECT ACTIONS', () => {
     });
   });
   it('Should not fail when creating a project as an admin.', () => {
-    cy.request({
+    cy.login(Cypress.env('admin_credentials'));
+    cy.authenticatedRequest({
       method: 'POST',
       url: '/api/projects',
       body: fakeProject,
-      auth: {
-        bearer: jwtAdmin,
-      },
-      failOnStatusCode: false,
     }).then((response) => {
       expect(response.status).to.eq(201);
       fakeProjectId = response.body.id;
     });
   });
   it('Should not fail when creating a project as an editor.', () => {
-    cy.request({
+    cy.login(Cypress.env('editor_credentials'));
+    cy.authenticatedRequest({
       method: 'POST',
       url: '/api/projects',
       body: fakeProject,
-      auth: {
-        bearer: jwtEditor,
-      },
-      failOnStatusCode: false,
     }).then((response) => {
       expect(response.status).to.eq(201);
     });
   });
 
   // READ TODO waiting for answers about granularity af reading rights.
   it('Should not fail when requesting projects.', () => {
+    cy.login(Cypress.env('editor_credentials'));
+    cy.authenticatedRequest({
+      method: 'GET',
+      url: '/api/projects',
+    }).then((response) => {
+      expect(response.status).to.eq(200);
+      expect(response.body).to.be.an('array');
+    });
+  });
+
+  it("A visitor mustn't see any draft projects", () => {
     cy.request({
       method: 'GET',
       url: '/api/projects',
-      auth: {
-        bearer: jwtEditor,
-      },
-      failOnStatusCode: false,
     }).then((response) => {
       expect(response.status).to.eq(200);
       expect(response.body).to.be.an('array');
+      response.body.forEach(({ draft }) => expect(draft).to.be.false);
     });
   });
+
   it('Should not fail when requesting a project.', () => {
-    cy.request({
+    cy.login(Cypress.env('editor_credentials'));
+    cy.authenticatedRequest({
       method: 'GET',
       url: `/api/projects/${fakeProjectId}`,
-      auth: {
-        bearer: jwtEditor,
-      },
-      failOnStatusCode: false,
     }).then((response) => {
       expect(response.status).to.eq(200);
       expect(response.body.id).to.eq(fakeProjectId);
     });
   });
 
-  // UODATE
+  // UPDATE
```

Comment:
Have you considered using describe block instead of comments for separating tests ?